src/evaluators/CallExpression.js

```diff
@@ -87,9 +87,6 @@ function callBothFunctionsAndJoinTheirEffects(
     // all the branches come together into one.
     completion = realm.composeWithSavedCompletion(completion);
   }
-
```

Comment:
Why delete the comment?

Reply:
I didn't delete it, it was still there. I force pushed the previous change. You were too ninja quick and saw before I got a chance to fix the push :D